internal/controller/variantautoscaling_controller.go

```diff
@@ -361,7 +425,7 @@ func (r *VariantAutoscalingReconciler) SetupWithManager(mgr ctrl.Manager) error
 	}
 
 	r.PromAPI = promv1.NewAPI(promClient)
-	logger.Log.Info("Prometheus client initialized")
+	logger.Log.Info("Prometheus client initialized successfully")
```

Comment:
How are we determiming success, we dont check error value, can you please check err val and determine if it's successful?